CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ OPTION(BUILD_BENCHMARKS
 
 
 # Find all dependencies here, and ensure they have IMPORTED targets
-
+LIST(APPEND CMAKE_PREFIX_PATH "/opt/wavelab")
```

Comment:
Can you make this an [option](https://cmake.org/cmake/help/v3.5/command/option.html)? The user should be able to disable adding this path.

You can also add a status message like "Adding /opt/wavelab to CMAKE_PREFIX_PATH", and a comment here that some of our  install scripts put stuff here. (it's not "all dependencies"). Just to avoid [surprising the user](https://en.wikipedia.org/wiki/Principle_of_least_astonishment)